README.md

```diff
@@ -60,7 +60,7 @@ It is setup with two primary machines: web and db. Together these two virtual ma
 4. Create a Magento 2 build available at m2.demo:
 
     ```bash
-    vagrant ssh web -- /server/vagrant/bin/m2setup.sh --sampledata --hostname=m2.demo
+    vagrant ssh web -- /server/vagrant/guest/bin/m2setup.sh --sampledata --hostname=m2.demo
```

Comment:
I ran this command to update my path 

PATH=$PATH:/server/vagrant/guest/bin 

but when I run 

vagrant ssh web -- m2setup.sh --sampledata --hostname=m3.demo

I still get 

bash: vagrant: command not found

What did I miss?